ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -250,6 +250,39 @@ def _metrics_tags(cls, span: Span) -> List[str]:
             tags.append("%s:%s" % (ERROR_TYPE, err_type))
         return tags
 
+    def extract_chat_completion_output_content(self, chat_completion: Any) -> Union[str, Dict[str, Any]]:
+        """
+        Extracts the completion content from a chat completion object.
+        This is handled differently for the different types of model providers
+        that can be used with LangChain.
+        """
+
+        try:
+            text = chat_completion.text
+            if text:
+                return text
+
+            message = chat_completion.message
+            content = message.content
+
+            tool_calls = message.tool_calls
+
+            # for anthropic, langchain puts it on the content
+            if content:
+                if isinstance(content, str):
+                    return content
```

Comment:
What happens if we have multiple content blocks in the `message.content` field? Looks like anthropic has the tool call is also exposed in `tool_calls`, but the content blocks in a `message.content` can contain an actual message (`content[0]["text"]`) as well as tool call arguments (i.e. `content[1]["input"]`) 
[ref](https://python.langchain.com/v0.2/docs/integrations/chat/anthropic/#content-blocks)

Reply:
good point! let me try this out.